migrations/1_create_asset_allocations_up.sql

```diff
@@ -0,0 +1,9 @@
+    CREATE TABLE asset_allocations(
+        asset_id varchar (100) ,
```

Comment:
why asset_id is varchar , if we are inserting string we have to ensure that its unique always. What's plan on that & why we ain't using serial number?